{
    "dataType": "CVE_RECORD",
    "dataVersion": "5.1",
    "cveMetadata": {
        "cveId": "CVE-2023-26112",
        "assignerOrgId": "bae035ff-b466-4ff4-94d0-fc9efd9e1730",
        "state": "PUBLISHED",
        "assignerShortName": "snyk",
        "dateReserved": "2023-02-20T10:28:48.922Z",
        "datePublished": "2023-04-03T05:00:01.443Z",
        "dateUpdated": "2024-08-02T11:39:06.600Z"
    },
    "containers": {
        "cna": {
            "metrics": [
                {
                    "cvssV3_1": {
                        "version": "3.1",
                        "attackVector": "NETWORK",
                        "attackComplexity": "HIGH",
                        "privilegesRequired": "NONE",
                        "userInteraction": "NONE",
                        "scope": "UNCHANGED",
                        "confidentialityImpact": "NONE",
                        "integrityImpact": "NONE",
                        "availabilityImpact": "LOW",
                        "baseScore": 3.7,
                        "baseSeverity": "LOW",
                        "vectorString": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:N/A:L/E:P"
                    }
                }
            ],
            "credits": [
                {
                    "value": "DarkTinia",
                    "lang": "en"
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "cweId": "CWE-1333",
                            "description": "Regular Expression Denial of Service (ReDoS)",
                            "lang": "en"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "orgId": "bae035ff-b466-4ff4-94d0-fc9efd9e1730",
                "shortName": "snyk",
                "dateUpdated": "2023-04-03T05:00:01.443Z"
            },
            "descriptions": [
                {
                    "value": "All versions of the package configobj are vulnerable to Regular Expression Denial of Service (ReDoS) via the validate function, using (.+?)\\((.*)\\).\r\r**Note:** This is only exploitable in the case of a developer, putting the offending value in a server side configuration file.\r\r",
                    "lang": "en"
                }
            ],
            "references": [
                {
                    "url": "https://security.snyk.io/vuln/SNYK-PYTHON-CONFIGOBJ-3252494"
                },
                {
                    "url": "https://github.com/DiffSK/configobj/issues/232"
                },
                {
                    "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/NZHY7B33EFY4LESP2NI4APQUPRROTAZK/"
                },
                {
                    "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/6BO4RLMYEJODCNUE3DJIIUUFVTPAG6VN/"
                },
                {
                    "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/PYU4IHVLOTYMFPH7KDOJGKZQR4GKWPFK/"
                }
            ],
            "affected": [
                {
                    "product": "configobj",
                    "versions": [
                        {
                            "version": "0",
                            "lessThan": "*",
                            "status": "affected",
                            "versionType": "semver"
                        }
                    ],
                    "vendor": "n/a"
                }
            ]
        },
        "adp": [
            {
                "providerMetadata": {
                    "orgId": "af854a3a-2127-422b-91ae-364da2661108",
                    "shortName": "CVE",
                    "dateUpdated": "2024-08-02T11:39:06.600Z"
                },
                "title": "CVE Program Container",
                "references": [
                    {
                        "url": "https://security.snyk.io/vuln/SNYK-PYTHON-CONFIGOBJ-3252494",
                        "tags": [
                            "x_transferred"
                        ]
                    },
                    {
                        "url": "https://github.com/DiffSK/configobj/issues/232",
                        "tags": [
                            "x_transferred"
                        ]
                    },
                    {
                        "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/NZHY7B33EFY4LESP2NI4APQUPRROTAZK/",
                        "tags": [
                            "x_transferred"
                        ]
                    },
                    {
                        "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/6BO4RLMYEJODCNUE3DJIIUUFVTPAG6VN/",
                        "tags": [
                            "x_transferred"
                        ]
                    },
                    {
                        "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/PYU4IHVLOTYMFPH7KDOJGKZQR4GKWPFK/",
                        "tags": [
                            "x_transferred"
                        ]
                    }
                ]
            }
        ]
    }
}